test/gtest/uct/test_ib.cc

```diff
@@ -442,7 +442,7 @@ UCS_TEST_P(test_uct_wakeup_ib, tx_cq)
     send_msg_e1_e2();
 
     /* make sure the file descriptor is signaled once */
-    ASSERT_EQ(poll(&wakeup_fd, 1, 10000), 1);
+    ASSERT_EQ(poll(&wakeup_fd, 1, 1000*ucs::test_time_multiplier()), 1);
```

Comment:
We had instabilities last couple weeks in tests which are waiting events on descriptors, so we need to understand which timeout works for any environment including several concurrent jobs on servers. `10000` was added in https://github.com/openucx/ucx/pull/1341, `1000*multiplier` was requested by @shssf and this is a kind of trying to adjust and unify the cases.